pixiedust/display/chart/renderers/matplotlib/barChartDisplay.py

```diff
@@ -57,17 +57,41 @@ def matplotlibRender(self, fig, ax):
                     index=keyFields[0], columns=clusterby, values=valueField
                 )
                 pivot.index.name=keyFields[0]
+
+                # sort order is lost after pivot
+                if self.options.get("sortby", None):
```

Comment:
You could probably avoid the second identical call to self.options.get("sortby", None)
e.g.
sortby = self.options.get("sortby", None)
if sortby:
   ...